core/src/main/java/org/elasticsearch/search/internal/InternalSearchHit.java

```diff
@@ -86,7 +82,7 @@
 
     private Map<String, HighlightField> highlightFields = null;
 
-    private Object[] sortValues = EMPTY_SORT_VALUES;
```

Comment:
seems like this empty array could be useful, shall we rather have a SearchSortValues.EMPTY default ?